app/utils/snack_bar/index.test.ts

```diff
@@ -15,4 +15,14 @@ describe('snack bar', () => {
             });
         });
     });
+
+    describe('showBoRPostErrorSnackbar', () => {
```

Comment:
Not testing that the custom message is properly sent through.

Reply:
Done.